geode-assembly/src/acceptanceTest/java/org/apache/geode/launchers/LocatorLauncherWithPulseAndCustomLogConfigAcceptanceTest.java

```diff
@@ -0,0 +1,268 @@
+        .directory(temporaryFolder.getRoot())
+        .command("java",
+            "-Dgemfire.http-service-port=" + httpServicePort,
+            "-Dgemfire.jmx-manager-start=true",
+            "-Djava.awt.headless=true",
+            "-Dlog4j.configurationFile=" + configWithoutGeodePluginsFile.toAbsolutePath(),
+            "-cp", geodeDependencies.toFile().getAbsolutePath(),
+            "org.apache.geode.distributed.LocatorLauncher", "start", LOCATOR_NAME,
+            "--port", String.valueOf(locatorPort));
+
+    System.out.println("Launching command: " + processBuilder.command());
+
+    locator = processBuilder
+        .start();
+
+    assertThat(locator.isAlive()).isTrue();
+
+    await().untilAsserted(() -> {
+      assertThat(locatorLogFile)
+          .doesNotExist();
+
+      assertThat(pulseLogFile)
+          .doesNotExist();
+
+      LogFileAssert.assertThat(stdoutFile.toFile())
+          .exists()
+          .contains("Located war: geode-pulse")
+          .contains("Adding webapp /pulse")
+          .contains("Starting server location for Distribution Locator")
+          .doesNotContain("geode-pulse war file was not found")
+          .doesNotContain("java.lang.IllegalStateException: No factory method found for class");
+    });
+  }
+
+  @Test
+  public void locatorLauncherUsesConfigFileInClasspathWithoutGeodePlugins() throws Exception {
+    copy(configWithoutGeodePluginsFile, temporaryFolder.getRoot().toPath().resolve("log4j2.xml"));
+
+    String classpath = temporaryFolder.getRoot().getAbsolutePath() + File.pathSeparator +
+        geodeDependencies.toFile().getAbsolutePath();
+
+    ProcessBuilder processBuilder = new ProcessBuilder()
+        .redirectErrorStream(true)
+        .redirectOutput(stdoutFile.toFile())
+        .directory(temporaryFolder.getRoot())
+        .command("java",
+            "-Dgemfire.http-service-port=" + httpServicePort,
+            "-Dgemfire.jmx-manager-start=true",
+            "-Djava.awt.headless=true",
+            "-cp", classpath,
+            "org.apache.geode.distributed.LocatorLauncher", "start", LOCATOR_NAME,
+            "--port", String.valueOf(locatorPort));
+
+    System.out.println("Launching command: " + processBuilder.command());
+
+    locator = processBuilder
+        .start();
+
+    assertThat(locator.isAlive()).isTrue();
+
+    await().untilAsserted(() -> {
+      assertThat(locatorLogFile)
+          .doesNotExist();
+
+      assertThat(pulseLogFile)
+          .doesNotExist();
+
+      LogFileAssert.assertThat(stdoutFile.toFile())
+          .exists()
+          .contains("Located war: geode-pulse")
+          .contains("Adding webapp /pulse")
+          .contains("Starting server location for Distribution Locator")
+          .doesNotContain("geode-pulse war file was not found")
+          .doesNotContain("java.lang.IllegalStateException: No factory method found for class");
+    });
+  }
+
+  @Test
+  @Ignore("GEODE-8197")
```

Comment:
My understanding is that once GEODE-8197 related bug is fixed, this `@Ignore` annotation will be removed.